client/App/pages/LandingPage/components/Navbar/Navbar.module.scss

```diff
@@ -0,0 +1,77 @@
+@import "/client/App/styles/utils";
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border-bottom: 1px solid #ebebeb;
+  font-size: 16px;
+
+  &__left {
+    background-color: $yellow;
+    padding: 15px 22px;
+
+    & img {
+      height: 40px;
+      width: 85px;
+    }
+  }
+
+  &__right {
+    padding-right: 15px;
+  }
+}
+
+.navigation {
+  &__list {
+    display: flex;
+  }
+
+  &__item {
+    >button {
```

Comment:
Don't use other selector than classes and ids, they are kinda unpredictable. This notation comes from bem: block__element. In good bem other selectors are unused and is nesting is limited to parent only. `
Good: parent: {&Child {}} 
Bad: parent: {&Child {&Child: {}}}
`